app/Services/Model/Imp/SummitOrderService.php

```diff
@@ -3609,10 +3609,16 @@ public function processSummitOrderReminders(Summit $summit): void
                 } catch (\Exception $ex) {
                     Log::error($ex);
                 }
+                $order_owner_id = $order->getOwner()->getId();
```

Comment:
this could throw a null exception
u need to check by email

Reply:
done